app/assets/v2/css/search_bar.css

```diff
@@ -110,7 +122,7 @@
   }
 
   .order_by .sort_option:last-child {
-    margin-right: 0px;
+    margin-right: 0;
```

Comment:
Is that what we follow ?
Didn't pay attention to this until now

Reply:
`px` for 0 is redundant. IntelliJ IDE gives a warning. Hence I remove them as it doesn't make sense to add `px` anyway.

Reply:
change IDE :P